### PR TITLE
[manuf] provision attestation key seed (output of DRBG) during personalization

### PR DESCRIPTION
**Pull request by timothytrippel**
This provisions the attestation keygen DRBG output seed into flash info page 4, in bank 0, partition 0. These seeds will be read out of flash and used by the `otbn_boot_services` lib to generate the attestation keys that will be FIPS 186-5 compliant.

### CODE REVIEW

**Comment by timothytrippel** [comment]
@jadephilipoom I ended up just writing the DRBG output to the flash info page, as it will be easier to forward along the various `attestation_seed`s to the `otbn_boot_attestation_keygen()` function. Lmk if this is ok, if I used the entropy complex cryptolib API correctly.

File: sw/device/silicon_creator/manuf/lib/personalize.c [hunk 1]
```diff
@@ -171,22 +172,75 @@ static status_t shares_check(uint64_t *share0, uint64_t *share1, size_t len) {
...
+ */
+OT_WARN_UNUSED_RESULT
+static status_t flash_attestation_key_seed_write(
+    dif_flash_ctrl_state_t *flash_state, flash_info_field_t field, size_t len) {
+  TRY(entropy_csrng_instantiate(/*disable_trng_input=*/kHardenedBoolFalse,
+                                /*seed_material=*/NULL));
+
+  uint32_t seed[kAttestationSeedWords];
+  TRY(entropy_csrng_generate(/*seed_material=*/NULL, seed, len,
+                             /*fips_check*/ kHardenedBoolTrue));
```
Comment (moidx) [inline sw/device/silicon_creator/manuf/lib/personalize.c:195]: nit: `/*fips_check=*/`

**Review by moidx — APPROVED** [review]

File: sw/device/silicon_creator/manuf/lib/personalize.c (hunk 1 above)
Comment (timothytrippel) [reply sw/device/silicon_creator/manuf/lib/personalize.c]: fixed.

**Comment by timothytrippel** [comment]
Closing this as commits were merged as part of #20255

**Pull request closed without merging**